rhcsa.html

```diff
@@ -463,7 +463,7 @@
 <p><span style="font-size: 10pt; font-family: &quot;Arial&quot;,&quot;sans-serif&quot;; color: black;">YOU
 HAVE ONE SYSTEM WITH ONE VIRTUAL SYSTEM IN BUILD<o:p></o:p></span></p>
 
-<p><span class="GramE"><span style="font-size: 10pt; font-family: &quot;Arial&quot;,&quot;sans-serif&quot;; color: black;">THE root password as "RedHat255".</span></span><span style="font-size: 10pt; font-family: &quot;Arial&quot;,&quot;sans-serif&quot;; color: black;"><o:p></o:p></span></p>
+<p><span class="GramE"><span style="font-size: 10pt; font-family: &quot;Arial&quot;,&quot;sans-serif&quot;; color: black;">THE root password as "<span style="color:#FF0000">RedHat255<span>".</span></span><span style="font-size: 10pt; font-family: &quot;Arial&quot;,&quot;sans-serif&quot;; color: black;"><o:p></o:p></span></p>
```

Comment:
Hey @sivakrishna-yarlagadda can we change hexa code with english names like instead of ```#00FFFF``` make it ```red```